apps/test/unit/code-studio/components/header/SignInCalloutTest.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import SignInCalloutLogic from '@cdo/apps/code-studio/components/header/SignInCalloutLogic';
+import {shallow} from 'enzyme';
+import {expect} from '../../../../util/reconfiguredChai';
+import i18n from '@cdo/locale';
+
+const wrapper = shallow(<SignInCalloutLogic />);
+
+describe('ViewPopup', () => {
```

Comment:
There should probably be another unit test for the wrapper where the display logic is tested

Reply:
I think all of the cookie/display testing will come in the next PR when it is actually connected, so this one only includes what the display looks like when it is displayed.